GVRf/Framework/src/org/gearvrf/animation/GVRAssimpAnimation.java

```diff
@@ -0,0 +1,37 @@
+/* Copyright 2015 Samsung Electronics Co., LTD
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gearvrf.animation;
+
+import org.gearvrf.GVRHybridObject;
+import org.gearvrf.GVRMesh;
+import org.gearvrf.GVRSceneObject;
+
+/** Animate Mesh from assimp **/
+public class GVRAssimpAnimation extends GVRAnimation {
```

Comment:
Assimp being "leaked". Assimp should be an implementation detail people are not aware of - we should strive for that at least.